doc/content/source/auxkernels/MaterialRealCMMAux.md

```diff
@@ -0,0 +1,32 @@
+# MaterialRealCMMAux
+
+!syntax description /AuxKernels/MaterialRealCMMAux
+
+## Description
+
+`MaterialRealCMMAux` can be used to populate an `AuxVariable` with an element of
+a material property that is of type, ColumnMajorMatrix (CMM). In MASTODON, a
+common use case is accessing the deformation or force CMMs from link elements
+when the [`LinearSpring`](source/materials/LinearSpring.md) material or any of
+the seismic isolator materials (e.g., [`ComputeFPIsolatorElasticity`](source/materials/ComputeFPIsolatorElasticity.md)) is used. This `AuxKernel` and the corresponding
```

Comment:
Does this mean we can directly get the isolator deformations without having to do a difference between top and bottom node displacements? If so, can you please state this explicitly so that the benefits of this auxkernel are clearer?